tests/utils/mod.rs

```diff
@@ -1,5 +1,5 @@
 use ayb::error::AybError;
-use quoted_printable;
```

Comment:
Wow! I must have left this behind and pushed the logic into the Pythonic SMTP stub server. Thank you!